charts/kserve-resources/templates/clusterservingruntimes.yaml

```diff
@@ -385,3 +385,34 @@ spec:
         limits:
           cpu: "1"
           memory: 2Gi
+---
+apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterServingRuntime
+metadata:
+  name: kserve-huggingface-triton-server
+spec:
+  annotations:
+    prometheus.kserve.io/port: '8080'
+    prometheus.kserve.io/path: "/metrics"
+  supportedModelFormats:
+    - name: huggingface
+      version: "1"
+      autoSelect: true
+      priority: 1
```

Comment:
I think the model format is actually triton in this case, triton does not support loading hugging face weights